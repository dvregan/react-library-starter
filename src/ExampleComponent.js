import React from "react";

const ExampleComponent = () => (
  <div>
    <h1>Example Component</h1>
    <p>
      This is an example component, edit <code>src/ExampleComponent.js</code> to
      get started.
    </p>
  </div>
);

export default ExampleComponent;
