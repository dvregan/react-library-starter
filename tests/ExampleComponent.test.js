import React from "react";
import renderer from "react-test-renderer";
import { ExampleComponent } from "../src";

describe("ExampleComponent", () => {
  it("renders the same snapshot", () => {
    const component = renderer.create(<ExampleComponent />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
