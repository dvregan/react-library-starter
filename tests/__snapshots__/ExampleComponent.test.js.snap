// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleComponent renders the same snapshot 1`] = `
<div>
  <h1>
    Example Component
  </h1>
  <p>
    This is an example component, edit 
    <code>
      src/ExampleComponent.js
    </code>
     to get started.
  </p>
</div>
`;
